views/scoredword.erb

```diff
@@ -0,0 +1,5 @@
+<% if @valid_word %>
+  <%= erb(:table) %>
```

Comment:
Good use of partials being rendered by other partials. For purposes of isolating "logic" from the view files I would recommend implementing these as methods in your Sinatra class:
```ruby
class MySite < Sinatra::Base
  def render_table
    erb :table
  end
  def render_error
    erb :error
  end
end
```

In this way you've isolated the knowledge from the view file that a specific ERB file named "table.erb" exists, and also that the partial should be rendered using ERB.